_config.yml

```diff
@@ -32,7 +32,6 @@ subtitle: Email government officials and council members to reallocate egregious
 meta: Email government officials and council members to reallocate police budgets
   towards education, social services, and dismantling racial inequality.
 footer_text: Want to add your city? [Open a request.](https://github.com/teddywilson/defund12.org/issues)
-email_intro_text: |-
-  This page automatically opens a drafted email for you.
-
-  Didn't work correctly? You can copy and paste the recipients, subject and message individually below.
+auto_open_message: This page automatically opens a drafted email for you.
+bad_mailto_message: Email link not working correctly? You can copy and paste the recipients,
+  subject and message individually below.
```

Comment:
Why was this changed?

Reply:
This seems to be related to Siteleaf's commit? I'll merge the latest changes into my fork and see if that resolves the issue.